### PR TITLE
Description in Japanese

### PR DESCRIPTION
**Pull request by KosukeMizuno**
Hi, I added an option for language select.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,12 @@ When you commit changes by vim as git default editor, this plugin automatically
 $ git commit
 ```
 
+You can choose language from english(default) or japanese.
```
Comment (gotchane) [inline README.md]:
```suggestion
You can choose the language for this plugin from English(default) or Japanese.
```

File: README.md [hunk 2]
```diff
@@ -43,6 +43,12 @@ When you commit changes by vim as git default editor, this plugin automatically
...
+```
+let g:git_commit_prefix_lang = 'en'  " or 'ja'
+```
+
+
```
Comment (gotchane) [inline README.md]: Please remove this additional new line.

File: autoload/git_commit_prefix.vim [hunk 3]
```diff
@@ -16,8 +16,25 @@ let s:git_commit_prefix_candidates = [
   \ {'word': 'perf: ', 'menu': 'A code change that improves performance'},
   \ {'word': 'test: ', 'menu': 'Adding missing or correcting existing tests'},
   \ {'word': 'chore: ', 'menu': 'Changes to the build process or auxiliary tools and libraries such as documentation generation'}]
+
+let s:git_commit_prefix_candidates_ja = [
+  \ {'word': 'feat: ', 'menu': '新機能'},
```
Comment (gotchane) [inline autoload/git_commit_prefix.vim:21]: I would like to set the prefix candidates to DRY(Don't Repeat Yourself), is that possible?
If it is difficult, it's OK to keep this implementation.

**Review by gotchane — COMMENTED** [review]
Sorry for late reply...
Thanks for your sending PR 👍 
I reviewed it.

File: README.md (hunk 2 above)
Comment (KosukeMizuno) [reply README.md]: I removed it.

File: autoload/git_commit_prefix.vim (hunk 3 above)
Comment (KosukeMizuno) [reply autoload/git_commit_prefix.vim:21]: I guess I can, but sorry, now I have no time to refactor it.

**Comment by KosukeMizuno** [comment]
I fixed it along with your comment.

**Comment by gotchane** [comment]
LGTM! Thanks 👍

**Pull request merged**